### PR TITLE
Refine examples in perf commands

### LINKED ISSUE #4605: [mesheryctl] `perf` examples are outdated
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
Users of `mesheryctl perf` are presented with the following output:

```
➜  ~ mesheryctl perf
Error: requires at least 1 arg(s), only received 0
Usage:
  mesheryctl perf [flags]
  mesheryctl perf [command]

Examples:

// Run performance test
mesheryctl perf apply --profile test --name \"a quick stress test\" --url http://192.168.1.15/productpage --qps 300 --concurrent-requests 2 --duration 30s --token \"provider=Meshery\"

// List performance profiles
mesheryctl perf profile

// List performance results
mesheryctl perf result


Available Commands:
  apply       Run a Performance test
  profile     List performance profiles
  result      List performance test results

Flags:
  -h, --help                   help for perf
  -o, --output-format string   (optional) format to display in [json|yaml]
  -t, --token string           (required) Path to meshery auth config

Global Flags:
      --config string   path to config file (default "/Users/lee/.meshery/config.yaml")
  -v, --verbose         verbose output

Use "mesheryctl perf [command] --help" for more information about a command.
```

The first example doesn't work and returns an error about an unknown token. Removing the token causes a second error about an unspecified profile name. 

#### Desired Behavior
Users should be able to copy and paste these examples and run them successfully.

#### Environment

- Host OS: Mac
- Meshery Server Version: stable-v0.5.71
- Meshery Client Version: stable-v0.5.71
- Platform: Kubernetes

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes #4605 

**Notes for Reviewers**
Do check and tell if token flag has to be added across all perf commands

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0293f2b14a0f16a427598bf312180592504187af at: https://618fd3edac6c7746e4c14c1a--meshery-docs-preview.netlify.app

**Comment by navendu-pottekkat** [comment]
@alphaX86 The perf command needs contextual errors (missing token --> run `mesheryctl system login`) and examples should be working. We shouldn't include the `--token` flag on the example as mesheryctl would by default use the default token.

**Comment by alphaX86** [comment]
> @alphaX86 The perf command needs contextual errors (missing token --> run `mesheryctl system login`) and examples should be working. We shouldn't include the `--token` flag on the example as mesheryctl would by default use the default token.
> 
> Instead please update the examples with a working one. I think in the examples the `token` flag does not have any values passed which might be causing the error in the issue opened.

Rog' will use with `mesheryctl system login` then instead of `--token`

**Review by piyushsingariya — COMMENTED** [review]
I am thinking about not adding the `-t`  in examples, we want people to be aware of the presence of auth login for meshery.

**Comment by navendu-pottekkat** [comment]
@piyushsingariya Yes. That is what I meant in the comment above.

**Comment by piyushsingariya** [comment]
> @piyushsingariya Yes. That is what I meant in the comment above.

mybad I didn't read it before!

**Comment by l5io** [comment]
🚀 Preview for commit 2df6fce1320f4cdd27a147f8330d93d2737b4d8c at: https://619515ce1f139112a5625c32--meshery-docs-preview.netlify.app

**Review by piyushsingariya — COMMENTED** [review]
@alphaX86 I think I failed to explain properly, We shouldn't add `mesheryctl auth login` in examples of perf, only demonstration of perf command is enough.

**Comment by alphaX86** [comment]
💆‍♂️💆‍♂️ I'll change it when free

--
Regards,
Aadhitya A

**Comment by leecalcote** [comment]
Thanks for this effort, @alphaX86 👍

**Comment by l5io** [comment]
🚀 Preview for commit ee1d7a64c8c7451fe326bb8786ce39455c71ae71 at: https://619722fc8d89ae58628dc031--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/perf/apply.go [hunk 1]
```diff
@@ -47,13 +47,22 @@ var applyCmd = &cobra.Command{
 mesheryctl perf apply meshery-profile --flags
 
 // Execute a Performance test with creating a new performance profile
-mesheryctl perf apply --profile meshery-profile-new --url <url>
+mesheryctl perf apply --profile meshery-profile-new --url "https://google.com"
 
-// Run Performance test using SMP compatible test configuration
-mesheryctl perf apply -f <filepath>
+// Run Performance test using SMP compatible test configuration (if available)
```
Comment (piyushsingariya) [inline mesheryctl/internal/cli/root/perf/apply.go:52]: `(if available)` why are we adding this here? I didn't get it.

Comment (alphaX86) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: Wait... isn't SMP configuration optional one? Normally users don't have SMP config file by default right? Or do they?

Comment (alphaX86) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: Correct me if I'm wrong 😨

Comment (piyushsingariya) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: It's not optional, it's kinda like a main feature. So I don't think we should promote this as optional.

Comment (piyushsingariya) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: It's not optional, it's kinda like a main feature. So I don't think we should promote this as optional.

Comment (alphaX86) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: 2 Times the msg is printed? 🤔 Ok bro, I'll drop that "optional" keyword... Any other changes I need to do or need to be aware of?

Comment (piyushsingariya) [reply mesheryctl/internal/cli/root/perf/apply.go:52]: > 2 Times the msg is printed?

yeah, I don't know why!

**Review by piyushsingariya — APPROVED** [review]

**Pull request merged**